.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,6 +1,12 @@
-# [TEMPLATE] {TITLE}
+---
+name: Bug Report
+about: Create a report to help us improve Near
```

Comment:
Maybe it would be nice to have something related to StatelessNet here, to signify that this template is meant for StatelessNet-specific issues.
For example: `Report a bug in StatelessNet`